Settings.hs

```diff
@@ -79,10 +79,12 @@ juliusFile = (if development then J.juliusFileReload
 
 data Extra = Extra
     { extraCopyright :: Text
+    , extraAllowAuthDummy :: Maybe Bool -- ^ Allow authDummy for development purposes
```

Comment:
This would more logically just be a `Bool`, and then below have `o .:? "allowAuthDummy" .!= False` to provide a default.